guides/v2.3/config-guide/cli/config-cli-subcommands-cache.md

```diff
@@ -73,9 +73,9 @@ You can clean (also referred to as _flush_ or _refresh_) cache types using eithe
 
 Command options:
 
-```terminal
-bin/magento cache:enable [type] ... [type]
-bin/magento cache:disable [type] ... [type]
+```bash
+   bin/magento cache:enable [type] ... [type]
+   bin/magento cache:disable [type] ... [type]
```

Comment:
Break lines 77 and 78 into separate command line examples.

Reply:
Done.